src/OpenSSL/SSL.py

```diff
@@ -2019,7 +2019,7 @@ def get_client_ca_list(self):
             result.append(pyname)
         return result
 
-    def makefile(self):
+    def makefile(self, mode=None, bufsize=None):
```

Comment:
Can you just have this accept `*args, **kwargs` since we don't care what args you call it with?